fastlane/Fastfile

```diff
@@ -936,5 +964,5 @@ def change_test_account_availability(availability)
     http.request(request)
   end
 
-  get_test_accounts_state
-end
+  UI.message("Accounts state after write: #{get_test_accounts_hash}")
+end
```

Comment:
Informational nitpick. 

With this symbol, GitHub is telling us that there is no new line at the end of the file.

![image](https://user-images.githubusercontent.com/1218433/120576978-f7523400-c466-11eb-8167-6b2c8ca37ead.png)

The convention of having a new line at the end of the files comes from an old C decision that has been passed down through Unix history:

> A source file that is not empty shall end in a new-line character, which shall not be immediately preceded by a backslash character.
> 
> Since this is a “shall” clause, we must emit a diagnostic message for a violation of this rule.

So, it turns out that, according to POSIX, every text file (including Ruby and JavaScript source files) should end with a `\n`, or “newline” (not “a new line”) character. This acts as the `eol`, or the “end of line” character. It is a line “terminator”. ([Source](https://thoughtbot.com/blog/no-newline-at-end-of-file))

**No need to address this now**, the next time someone with an editor configured to fix this will touch the file, it'll go away. 👍